### DISCUSSION TITLE
Consigli per nuova casa (sonoff, shelly, HASSIO)

### DISCUSSION THREAD

**Discussion by Ancalgon** [indomus/forum#907] [Hardware & Firmware]
Buongiorno a tutti, ho bisogno di qualche consiglio da parte di chi ne capisce più di me. Sto usando gia Home Assistant da un annetto e tra un mesetto arriverà l'elettricista per costruire l'impianto elettrico della mia nuova casa (in cui non abito ancora). In questo momento nel mio appartamento ho delle luci smart e funzionano a dovere se non per due grossi problemi che voglio risolvere e non portarmi nella nuova casa: interruttori e mancanza elettrica. 

INTERRUTTORI E CONTROLLO: adesso usiamo l'app android oppure degli switch ikea per accendere le luci. Per la nuova casa vorrei degli switch più seri, tipo quelli della sonoff per intenderci con tre pulsanti. Però ho dei dubbi e mi chiedo come muovermi. Meglio prendere prendere un sonoff (o shelly) da mettere dietro un classico interruttore oppure prendere direttamente un interruttore smart tipo i sonoff? Necessitano del firmware tasmota o vanno bene così? Ci sono alternative?

MANCANZA ELETTRICA: un grosso problema che abbiamo riscontrato in un anno è l'automatismo delle lampadine a riaccendersi dopo una mancanza elettrica. Visto il fastidio di svegliarsi alle 3 del mattino con la casa accesa, vorrei evitare questo problema. Ho letto che la colpa dovrebbe essere della lampadina stessa, non so se è così però. Consigli?

EXTRA: vorrei delle apparechiature (smart switch, sonoff o shelly) che misurino i consumi, sono tutti abili a farlo?
Dato che non vorrei fare flash strani, siamo ancora costretti a flasshare il firmware tasmota sui vari dispositivi shelly o sonoff?
Domanda banale, se acquisto uno switch sonoff per controllare le luci, non necessito un sonoff dietro per controllarlo vero? Oppure si.

**Comment by Scialla** [comment]
- se vuoi roba seria meglio andare su serie "nostrane" tipo bticino o vimar, serie domotiche
- se vuoi risparmiare vai pure sui sonoff, rigorosamente da flashare con tasmota o con esphome altrimenti, per accendere una luce, passi dalla cina... svincolati dai cloud dei produttori
- se preferisci usare shelly o similari, dietro gli interruttori, tieni presente che nella 503 ce ne sta uno e se piloti tre luci non riesci, devi mettere i device nelle cassette di derivazione o, meglio, a quadro.
- il problema delle lampadine quando manca la corrente è comune; o hanno la possibilità di una impostazione (power outage o similari) per impostare lo stato in caso di mancanza di corrente, oppure devi fare una automazione all'avvio di HA (se manca la corrente pure a lui) che spenga la lampadina
- se non vuoi flashare usa prodotti zigbee, ne esistono che misurano anche i consumi
- per l'ultima domanda la risposta è no

**Comment by Ancalgon** [comment]
Ho letto che alcuni modelli sonoff (gli ultimi usciti) non necessitano di flash tasmota per lavorare fuori cloud (cosa tassativa per me), ma credo che gli switch siano esclusi e necessitino il flash. Qualcuno conferma? 
Non essendo ferrato, meglio installare dei sonoff mini dietro le singole prese/interruttori oppure installare 2-3 "controller" direttamente nel quadro?
Il problema del power outage meglio gestirlo a livello sonoff oppure a livello lampadina? Quindi devo stare attendo che il sonoff abbia questa possibilità oppure cercare lampadine che mi permettano di controllare questo stato?
Tutte le lampadine che ho in uso sono zigbee, ma da quanto ho trovato i sonoff sono tutti wifi, idem i shelly, sbaglio?

**Reply by Scialla** [reply]
- i device sonoff li puoi pilotare localmente con integrazione sonoffLAN, ma hai molte possibilità in meno
- scelta tua, tieni presente che nella 503 lo spazio è esiguo 
- se gestibile sulla lampadina direttamente è ovviamente meglio; molte lampade e prese zigbee haoo questa possibilità, lo vedi già sulla webui
- sonoff fa device wifi e zigbee, shelly solo wifi
